recipes/influxdb-cpp-rest/1.0.1/conanfile.py

```diff
@@ -0,0 +1,78 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import get
+import os
+
+required_conan_version = ">=1.54.0"
+
+
+class InfluxdbCppRestConan(ConanFile):
+    name = "influxdb-cpp-rest"
+    description = "A C++ client library for InfluxDB using C++ REST SDK"
+    topics = ("influxdb", "cpprest", "http", "client")
+    license = "MPL-2.0"
+    homepage = "https://github.com/d-led/influxdb-cpp-rest"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            self.options.rm_safe("fPIC")
+
+    def configure(self):
+        if self.options.shared:
+            self.options["cpprestsdk"].shared = True
+
+    def requirements(self):
+        self.requires("cpprestsdk/2.10.19")
+        self.requires("rxcpp/4.1.1")
+
+    def build_requirements(self):
+        # Only for tests - not linked to the library
+        self.test_requires("catch2/3.11.0")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        deps = CMakeDeps(self)
+        deps.generate()
+        tc = CMakeToolchain(self)
+        # Disable tests and demo for packaging
+        tc.variables["BUILD_TESTING"] = False
+        tc.variables["BUILD_DEMO"] = False
```

Comment:
```suggestion
        tc.cache_variables["BUILD_TESTING"] = False
        tc.cache_variables["BUILD_DEMO"] = False
```

`cache_variables` are provided via `-D` flags as users would, whereas `variables` are in-line defined in the loaded `conan_toolchain.cmake` - so we probably want cache variables here